torch/optim/lr_scheduler.py

```diff
@@ -458,7 +483,7 @@ class StepLR(LRScheduler):
     """Decays the learning rate of each parameter group by gamma every step_size epochs.
 
     Notice that such decay can happen simultaneously with other changes to the learning rate
-    from outside this scheduler. When last_epoch=-1, sets initial lr as lr.
+    from outside this scheduler. When last_epoch=-1, use optimizer's lr as inital lr.
```

Comment:
```suggestion
    from outside this scheduler. When last_epoch=-1, use optimizer's lr as initial lr.
```

What if last_epoch is not -1? Then what's the inital lr?